pycode/memilio-epidata/memilio/epidata/assessNPIEffects.py

```diff
@@ -306,68 +314,73 @@ def analyze_npi_data(
         corr_pairwdist = hierarchy.distance.pdist(
```

Comment:
[scipy.spatial.distance.pdist](https://docs.scipy.org/doc/scipy/reference/generated/scipy.spatial.distance.pdist.html#scipy.spatial.distance.pdist) seems to be the new name